lib/cosmos/models/target_model.rb

```diff
@@ -266,21 +266,30 @@ def update_store(temp_dir)
       @id = target.id
       update()
 
-      # Load Packet Definitions
+      # Store Packet Definitions
       system.telemetry.all.each do |target_name, packets|
-        Store.instance.del("#{@scope}__cosmostlm__#{target_name}")
+        Store.del("#{@scope}__cosmostlm__#{target_name}")
         packets.each do |packet_name, packet|
           Logger.info "Configuring tlm packet: #{target_name} #{packet_name}"
-          Store.instance.hset("#{@scope}__cosmostlm__#{target_name}", packet_name, JSON.generate(packet.as_json))
+          Store.hset("#{@scope}__cosmostlm__#{target_name}", packet_name, JSON.generate(packet.as_json))
         end
       end
       system.commands.all.each do |target_name, packets|
-        Store.instance.del("#{@scope}__cosmoscmd__#{target_name}")
+        Store.del("#{@scope}__cosmoscmd__#{target_name}")
         packets.each do |packet_name, packet|
           Logger.info "Configuring cmd packet: #{target_name} #{packet_name}"
-          Store.instance.hset("#{@scope}__cosmoscmd__#{target_name}", packet_name, JSON.generate(packet.as_json))
+          Store.hset("#{@scope}__cosmoscmd__#{target_name}", packet_name, JSON.generate(packet.as_json))
         end
       end
+      # Store Limits Groups (which are already a Hash)
+      Store.hset("#{@scope}__cosmos_system", 'limits_groups', JSON.generate(system.limits.groups))
+      # Merge in Limits Sets
+      sets = Store.hget("#{@scope}__cosmos_system", 'limits_sets')
+      sets = JSON.parse(sets) if sets
+      sets ||= []
+      sets.concat(system.limits.sets.map(&:to_s)) # Convert the symbols to strings
+      Store.hset("#{@scope}__cosmos_system", 'limits_sets', JSON.generate(sets.uniq)) # Ensure uniq set
+
```

Comment:
This new block of code updates the Redis limits groups and sets based on the parsed telemetry